### DISCUSSION TITLE
Restart via mqtt does not reload the config

### DISCUSSION THREAD

**Discussion by grafalex82** [Koenkk/zigbee2mqtt#6628] [General]
I noticed that zigbee2mqtt/bridge/request/restart command restarts the z2m, but changes in the configuration are not reloaded. Particularly after setting a friendly name to a device, and restarting z2m via mqtt message, z2m continues to send my device messages at the old name. 

I have to restart whole docker container to force z2m pick configuration changes.

Is this a bug or expected behavior? How can I force reloading the config?

**Comment by Koenkk** [comment]
Should be fixed now, can you confirm?

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Reply by grafalex82** [reply]
Looks like fixed on latest-dev. 

Tested with this scenario:
- paired a switch
- changed friendly name
- restarted z2m via mqtt
- switch clicks are reported with friendly name in console

Thank you for fixing that
